content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -39,31 +39,37 @@ model where `Pods` can be treated much like VMs or physical hosts from the
 perspectives of port allocation, naming, service discovery, load balancing,
 application configuration, and migration.
 
+Kubernetes IP addresses exist at the `Pod` scope - containers within a `Pod`
+share their network namespaces - including their IP address and MAC address.
+This means that containers within a `Pod` can all reach each other's ports on
+`localhost`. This also means that containers within a `Pod` must coordinate port
+usage, but this is no different from processes in a VM.  This is called the
+"IP-per-pod" model.
+
+In every cluster, there exists an abstract pod-network to which pods
+are connected by default, unless explicitly configured to use the
+host-network (which is an optional capability). Even if a host have
+multiple IPs, host-network pods only have one Kubernetes IP address at
+the `Pod` scope, so the "IP-per-pod" model is guaranteed.
+
 Kubernetes imposes the following fundamental requirements on any networking
 implementation (barring any intentional network segmentation policies):
 
-   * pods on a node can communicate with all pods on all nodes without NAT
-   * agents on a node (e.g. system daemons, kubelet) can communicate with all
-     pods on that node
+   * pod-network pods on a node can communicate with all pod-network pods on
+     all nodes without NAT
+   * non-pod processes on a node (e.g. system daemons, kubelet) can communicate
+     with all pods on that node
 
-Note: For those platforms that support `Pods` running in the host network (e.g.
-Linux):
+Note: For those platforms that support `Pods` running in the host network (e.g.Linux):
 
-   * pods in the host network of a node can communicate with all pods on all
-     nodes without NAT
+   * host-network pods of a node can connect directly with all pods IPs
```

Comment:
Above you left it as "can communicate with" but here changed it to "can connect directly with".  That seems like a very specific change and I'd like to understand why?  My initial feeling is that we should use the same words in both clauses.

Also a nit:  "communicate with" or "connect to".  English :shrug:

Reply:
while we're nitpicking prepositions, "pods of a node" sounds weird to me and I think "pods on a node" would be better...

Reply:
I don't know how to express direction and origin, 
pod.Status.PodIPs should be reachable ALWAYS, but for host-network pod the origin can be any of the host IP addresses, that is not necessarily published in the API